### PR TITLE
Add expression graphs for NLP tests

### PR DESCRIPTION
**Pull request by odow**

### CODE REVIEW

File: src/Test/nlp.jl [hunk 1]
```diff
@@ -174,12 +188,22 @@ end
...
-        return [:Grad, :Jac, :Hess]
+        return [:Grad, :Jac, :Hess, :ExprGraph]
     else
-        return [:Grad, :Jac]
+        return [:Grad, :Jac, :ExprGraph]
     end
 end
 
+function MOI.objective_expr(d::FeasibilitySenseEvaluator)
+    return :(0 * x[1])
```
Comment (mlubin) [inline src/Test/nlp.jl]: This seems hacky. What about returning 0.0 instead?

Comment (odow) [reply src/Test/nlp.jl]: I think there is a bug somewhere if you pass 0.0. It's on my list of things to investigate today.

File: src/Test/nlp.jl [hunk 2]
```diff
@@ -174,12 +188,22 @@ end
...
-        return [:Grad, :Jac, :Hess]
+        return [:Grad, :Jac, :Hess, :ExprGraph]
     else
-        return [:Grad, :Jac]
+        return [:Grad, :Jac, :ExprGraph]
     end
 end
 
+function MOI.objective_expr(d::FeasibilitySenseEvaluator)
+    return :(0.0)
```
Comment (mlubin) [inline src/Test/nlp.jl]: `:(0.0)` parses exactly the same as `0.0`.

File: src/Test/nlp.jl (hunk 1 above)
Comment (chriscoey) [reply src/Test/nlp.jl]: worth merging in master and trying again?

**Pull request closed without merging**